src/Exceptions/UrlGenerationException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Laravel\Folio\Exceptions;
+
+use Exception;
+
+class UrlGenerationException extends Exception
+{
+    public static function forMissingParameter(string $path, string $parameter): static
```

Comment:
Can you add a simple comment to this method, like we do in any other methods in this project?